.evergreen/config.yml

```diff
@@ -157,6 +158,8 @@ functions:
           SSL=${SSL} \
           AUTH=${AUTH} \
           SWIFT_VERSION=${SWIFT_VERSION} \
+          MONGODB_API_VERSION=${MONGODB_API_VERSION} \
+          TEST_FILTER=${TEST_FILTER} \
```

Comment:
this wasn't strictly necessary, as we could have just run the whole test suite with `acceptAPIVersion2` on, but it seemed redundant to do so.